### PR TITLE
Performance upgrade and backward compatibility for new detail page

### PR DESCRIPTION
**Pull request by sanbornsen**
* Stopped loading of lists on navigate to detail page (Observable handle issue fixed)
* Temporary code added to redirect to the new detail page from the old detail page

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2131-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2131-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@sanbornsen fabric8/fabric8-ui:SNAPSHOT-PR-2131-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2131-fabric8-planner.badger.fabric8.io

File: src/app/components/work-item-new-detail/work-item-new-detail.component.ts [hunk 1]
```diff
@@ -165,7 +184,7 @@ export class WorkItemNewDetailComponent implements OnInit, OnDestroy {
           this.loadingIteration = true;
           this.loadingArea = true;
         })
-        .switchMap(() => this.workItemService.getWorkItemById(id, owner, space))
+        .switchMap(() => this.workItemService.getWorkItemByNumber(id, owner, space))
         .do(workItem => {
```
Comment (nimishamukherjee) [inline src/app/components/work-item-new-detail/work-item-new-detail.component.ts]: On the PR snapshot - this still fails. The URL displays the system number but GET call fails to load all the details.

Comment (sanbornsen) [reply src/app/components/work-item-new-detail/work-item-new-detail.component.ts]: I have tried the second snapshot and worked fine for me.

**Pull request merged**